src/main/java/frc/robot/subsystems/CommandSwerveDrivetrain.java

```diff
@@ -310,6 +310,10 @@ public double getAngle() {
         return this.getRotation2d().getDegrees();
     }
 
+    public boolean isAtSetpoint() {
```

Comment:
Let's change the name of this method to make it clear that it's at the **rotation** setpoint